src/CacheItemInterface.php

```diff
@@ -72,12 +72,12 @@ public function isHit();
      * @return static
      *   The invoked object.
      */
-    public function set($value);
+    public function set(mixed $value);
 
     /**
      * Sets the expiration time for this cache item.
      *
-     * @param \DateTimeInterface|null $expiration
+     * @param ?\DateTimeInterface $expiration
```

Comment:
Should stay as is: https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html

Reply:
cf https://github.com/php-fig/cache/pull/24#discussion_r543536544